CODE_OF_CONDUCT.md

```diff
@@ -1,13 +1,21 @@
-# Contributor Code of Conduct
+# Code of Conduct
 
-As contributors and maintainers of this project, we pledge to respect all people who contribute through reporting issues, posting feature requests, updating documentation, submitting pull requests or patches, and other activities.
+### We pledge to respect everyone who contributes to this project
 
-We are committed to making participation in this project a harassment-free experience for everyone, regardless of level of experience, gender, gender identity and expression, sexual orientation, disability, personal appearance, body size, race, ethnicity, age, or religion.
+As contributors and maintainers of this project, we pledge to respect everyone who contributes by reporting issues, requesting features, updating documentation, submitting pull requests, or anything else to help the project.
 
-Examples of unacceptable behavior by participants include the use of sexual language or imagery, derogatory comments or personal attacks, trolling, public or private harassment, insults, or other unprofessional conduct.
+##### This is a harassment-free zone
 
-Project maintainers have the right and responsibility to remove, edit, or reject comments, commits, code, wiki edits, issues, and other contributions that are not aligned to this Code of Conduct. Project maintainers who do not follow the Code of Conduct may be removed from the project team.
+We are committed to making participation in this project a harassment-free experience for everyone, regardless of level of experience, gender identity and expression, sexual orientation, disability, personal appearance, body size, race, ethnicity, age, or religion.
+
+Examples of unacceptable behavior include the use of sexual language or imagery, derogatory comments or personal attacks, trolling, public or private harassment, insults, or other unprofessional conduct.
+
+##### Project moderation
```

Comment:
Whatcha think about making this subheading `Project maintainers enforce conduct`? That sums up the section, and makes the subhead skimmable like the others